recipe/meta.yaml

```diff
@@ -49,6 +49,9 @@ requirements:
     - python
     - czmq
     - {{ mpi }}
+    - cffi
+    - ply
+    - lua-luaposix
```

Comment:
Can you clarify the difference between `host` and `run`? e.g. also libarchive, libhwloc, sqlite, zeromq, etc are required at runtime if this what the `run` tag indicates.

I good test would be to install this flux-core package if possible and try `flux start -s 4 flux run -N4 hostname` which exercises not all but many parts of the code, including the Python requirements.

Reply:
Regarding the `host` vs `run`: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run 
Still it also depends on the `run_exports` in the other packages, if those are missing for `shared libraries` then we have to add them as well. I try to push those changes up-stream but that might take some time, so in the mean time I added them here. In terms of the tests, that is something I plan to do at the end of the up-coming week. It is all a bit busy right now.

Reply:
Ah, ok, that is starting to make sense. Thanks!